my-inner-goddess

```diff
@@ -1,2 +1 @@
- “My inner goddess is jumping up and down, clapping her hands like a five year old.” 
```

Comment:
🍆🍆🍆 But now I feel like a receptacle -- an empty vessel to be filled at his whim.